AdobeStockAsset/etc/db_schema.xml

```diff
@@ -0,0 +1,129 @@
+        <column xsi:type="int" name="width" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Width"/>
+        <column xsi:type="int" name="height" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Height"/>
+        <column xsi:type="varchar" name="url" length="255" nullable="true" comment="Url"/>
+        <column xsi:type="varchar" length="255" name="country_name" nullable="true" comment="Country Name"/>
+        <column xsi:type="varchar" length="255" name="details_url" nullable="true" comment="Details Url"/>
+        <column xsi:type="varchar" length="255" name="vector_type" nullable="true" comment="Vector Type"/>
+        <column xsi:type="varchar" length="255" name="content_type" nullable="true" comment="Content Type"/>
+        <column xsi:type="varchar" length="255" name="creation_date" nullable="true" comment="Creation Date"/>
+        <column xsi:type="timestamp" name="created_at" on_update="true" nullable="false" default="CURRENT_TIMESTAMP" comment="Created At"/>
+        <column xsi:type="timestamp" name="updated_at" on_update="true" nullable="false" default="CURRENT_TIMESTAMP" comment="Updated At"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_ASS_MED_TYP_ID" table="adobe_stock_asset" column="media_type_id" referenceTable="adobe_stock_media_type" referenceColumn="id" onDelete="SET NULL"/>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_ASS_CAT_ID" table="adobe_stock_asset" column="category_id" referenceTable="adobe_stock_category" referenceColumn="id"  onDelete="SET NULL"/>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_ASS_CRE_ID" table="adobe_stock_asset" column="creator_id" referenceTable="adobe_stock_creator" referenceColumn="id" onDelete="SET NULL"/>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_ASS_PRE_LEV_ID" table="adobe_stock_asset" column="premium_level_id" referenceTable="adobe_stock_premium_level" referenceColumn="id" onDelete="SET NULL"/>
+    </table>
+    <table name="adobe_stock_category" resource="default" engine="innodb" comment="Adobe Stock Category">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="int" name="adobe_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Adobe Id"/>
+        <column xsi:type="varchar" length="255" name="name" nullable="true" comment="Name"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="id"/>
+        </constraint>
+    </table>
+    <table name="adobe_stock_asset_keyword" resource="default" engine="innodb" comment="Adobe Stock Asset Keyword">
```

Comment:
Foreign key constraints to keyword and asset tables need to be added to this table

Reply:
Hi @sivaschenko I am not sure how to add this change.
I mean, there is already a foreign key constraints in table `adobe_stock_asset` line 78 referencing the `category_id` column in this table, but there are not references to category in `adobe_stock_asset_keyword` nor `adobe_stock_keyword`

Reply:
@diazwatson I think I confused you by not referencing the table names in my comments, sorry for that.

I meant that it would be nice to have foreign key constraints for `adobe_stock_asset_keyword` to `adobe_stock_asset` and `adobe_stock_keyword` tables. But let's probably leave this for an additional pull request as a lot of work has already been done in this one.